Shared/Samples/Show portal user info/README.md

```diff
@@ -0,0 +1,42 @@
+# Show portal user info
+
+Retrieve a user's details via a Portal.
+
+![Image of show portal user info sample](show-portal-user-info.png)
+
+## Use case
+
+This portal information can be used to provide a customized UI experience for the user. For example, you can show a thumbnail of the user in the application to indicate that they are currently logged in.
+
+## How to use the sample
+
+Enter your ArcGIS Online credentials for the specified URL.
+
+## How it works
+
+1. On startup, the app presents the user with an editable text field containing a portal URL.
+2. Upon pressing the "Load" button, a portal will be created and loaded.
+3. If the portal is secured, it may potentially issue an authentication challenge.
+4. If the portal is successfully loaded, the info screen below will display the portal info, otherwise it will display the loading error.
```

Comment:
I glanced through some of the other READMEs, looking for mentions of how an app might handle things. For the ones I did read, they seem to be written in an active voice and use only what happens in the context of the sample.

```suggestion
1. Present the user with an editable text field for enterting a portal URL.
2. Create and load a portal with the URL.
3. If the portal is secured, it may potentially issue an authentication challenge.
4. Display the desired portal info once loading has successfully completed. Otherwise, if loading failed, display the error.
```